marimbabot_vision/scripts/flatten_whiteboard_data.py

```diff
@@ -0,0 +1,25 @@
+import os
+import tqdm
+from multiprocessing import Pool
+import re
+from shutil import copyfile
+
+NUM_WORKER = 30
```

Comment:
The other scripts are now being migrated to argparse, maybe we want to do this here too.

Reply:
Done. Looks a lot fancier now :P